aggs_metrics_percentile_ranks.go

```diff
@@ -1,25 +1,32 @@
+// Copyright 2012-present Oliver Eilhard. All rights reserved.
+// Use of this source code is governed by a MIT-license.
+// See http://olivere.mit-license.org/license.txt for details.
+
 package aggretastic
 
-import "github.com/olivere/elastic"
+import (
+	"github.com/olivere/elastic"
+)
 
 // PercentileRanksAggregation
 // See: https://www.elastic.co/guide/en/elasticsearch/reference/6.2/search-aggregations-metrics-percentile-rank-aggregation.html
 type PercentileRanksAggregation struct {
-	*tree
-
-	field       string
-	script      *elastic.Script
-	format      string
-	meta        map[string]interface{}
-	values      []float64
-	compression *float64
-	estimator   string
+	field		string
+	script		*elastic.Script
```

Comment:
Comment is actual for all aggregations that comes from olivere and have `*Script` or `*Query` or  other olivere dependencies. I'm just writing this comment on a random aggregation:

I assume from your code you do two main modifications with every source file: 
1) you change the package name and add the `injectable` logic (embed the struct, etc)
2) you fix the olivere types (all `*Script` are breaking and needs to be changed to `*elastic.Script`)

Right?
When I run `sync` process a few times, I've noticed there maybe lots of message that `Script` or `Query` is not found. Maybe because action(1) and action(2) need to be done together?

Also, I got millions of these (the same file and line were logged):
```
Fix procedure on: tmp/build_tmp/aggs-injectable.go:4:2: could not import github.com/olivere/elastic (can't find import: "github.com/dkonovenschi/aggretastic-1/vendor/github.com/olivere/elastic")
```

Reply:
Every "fix procedure" message is a notice about running fix procedure process of an error, which wrote after "fix procedure:". 
About millions of messages. "go/types" importer works badly with "vendor" imports. Try to delete "vendor" directory before running sync